binaries/runtime/src/operator/mod.rs

```diff
@@ -9,17 +9,17 @@ mod shared_lib;
 
 pub struct Operator {
     operator_task: Sender<OperatorInput>,
-    config: OperatorConfig,
+    config: OperatorDefinition,
 }
 
 impl Operator {
     pub async fn init(
-        operator_config: OperatorConfig,
+        operator_config: OperatorDefinition,
```

Comment:
```suggestion
        operator_definition: OperatorDefinition,
```

In the same way, `s/operator_config/operator_definition`